library/psa_crypto_ecp.c

```diff
@@ -96,6 +97,17 @@ psa_status_t mbedtls_psa_ecp_load_representation(
         goto exit;
     }
 
+    /* Get the exact number of bits which are necessary for this key. This is
+     * used to validate the "curve_bits" input parameter (only in case it was
+     * provided).
+     * Note: we intentionally ignore the return value of mbedtls_ecc_group_to_psa()
+     *       because we are only interested in the curve's bit size. */
```

Comment:
Perhaps also tell static analyzers: `(void) mbedtls_ecc_group_to_psa(...)`. Coverity for instance tends to warn when the return value is ignored in one call while is usually isn't. (Edit: removed in a later commit.)